pkg/agent/evictionmanager/manager.go

```diff
@@ -45,16 +45,19 @@ import (
 	"github.com/kubewharf/katalyst-core/pkg/consts"
 	"github.com/kubewharf/katalyst-core/pkg/metaserver"
```

Comment:
for now, only eviction-manager will call the auth functions (but not for reporter .etc), right?